README.md

```diff
@@ -116,6 +116,7 @@ The API serves the following endpoints:
 *Related to checks:*
 - `GET` on `/api/checks/latest/?url={url}&resource_id={resource_id}` to get the latest check for a given URL and/or `resource_id`
 - `GET` on `/api/checks/all/?url={url}&resource_id={resource_id}` to get all checks for a given URL and/or `resource_id`
+- `GET` on `/api/checks/aggregate/?group_by={column}&created_at={date}` to get checks occurences grouped by a `column` for a specific `date`
```

Comment:
What about `/api/checks?group_by={column}&created_at={date}` instead? Maybe it is as explicit?
Isn't the aggregation an internal mecanism which doesn't need to be explicited in the URL?
Or, do we need to have an `aggregate` in the URL to differentiate other future type of search parameters?
Just raising the question, but might be more difficult to separate the routes in the code that way.

Reply:
I wanted to start with this approach, but I had some considerations:
* the payload would vary greatly depending on `group_by` being used or not.
* It also does not actually return `checks` objects per se, it returns count aggregation.
* Since we did not have any existing `GET /api/checks`, it seemed weird to require a `group_by` parameter on this route. It made me think that `/api/checks/latest` and `/api/checks/all` should probably be `/api/checks` with additional relevant params (pagination would do actually). Thus going with `/aggregate` path for now. A refactoring may follow to standardize `/api/checks` routes maybe.

Reply:
Fair enough then, let's keep it that way. Thanks for the explanation!

Reply:
I am leaving it unresolved as it may be useful reference for future readers.